### PR TITLE
Ddcnl 7535: refactor/add toggles

### PR DESCRIPTION
**Pull request by SandeepUmredkar**

### CODE REVIEW

File: app/controllers/HomeController.scala [hunk 1]
```diff
@@ -89,7 +89,7 @@ class HomeController @Inject() (
                                                                                                  taxCalculationStateCyMinusOne,
                                                                                                  taxCalculationStateCyMinusTwo
                                                                                                )
-            shutteringMessaging                                                             <- featureFlagService.get(NpsShutteringToggle)
+            shutteringMessaging                                                             <- featureFlagService.get(ShowNpsOutageBannerToggle)
```
Comment (pascal22p) [inline app/controllers/HomeController.scala:92]: rename ShowNpsOutageBannerToggle to ShowOutageBannerToggle

File: app/models/admin/FeatureFlags.scala [hunk 2]
```diff
@@ -144,3 +140,31 @@ case object BreathingSpaceIndicatorToggle extends FeatureFlagName {
     "Enable/disable calls to Breathing space"
   )
 }
+
+case object AddressJourneyEnforcerToggle extends FeatureFlagName {
+  override val name: String                = "address-journey-enforcer-toggle"
+  override val description: Option[String] = Some(
+    "Enable/disable call to for address journey"
```
Comment (pascal22p) [inline app/models/admin/FeatureFlags.scala:147]: Is this the change of address or there is more?
"Enable/disable the change of address journey"

File: app/models/admin/FeatureFlags.scala [hunk 3]
```diff
@@ -144,3 +140,31 @@ case object BreathingSpaceIndicatorToggle extends FeatureFlagName {
...
+}
+
+case object DfsDigitalFormFrontendShutteredToggle extends FeatureFlagName {
+  override val name: String                = "dfs-digital-forms-frontend-shuttered-toggle"
+  override val description: Option[String] = Some(
+    "Enable/disable indicating whether the dfs digital forms frontend is shuttered"
+  )
+}
+
+case object AddressChangeAllowedToggle extends FeatureFlagName {
```
Comment (pascal22p) [inline app/models/admin/FeatureFlags.scala:165]: What's the difference with AddressJourneyEnforcerToggle?

Comment (pascal22p) [reply app/models/admin/FeatureFlags.scala:165]: The logic is opposite to the other toggles above. i.e. when all is well this one is set to false while the other are set to true.

File: app/models/admin/FeatureFlags.scala [hunk 4]
```diff
@@ -93,23 +96,16 @@ case object TaxcalcMakePaymentLinkToggle extends FeatureFlagName {
   override val description: Option[String] = Some("Enable/disable direct link to make a payment on taxcalc tile")
 }
 
-case object NpsShutteringToggle extends FeatureFlagName {
-  override val name: String                = "nps-shuttering-toggle"
-  override val description: Option[String] = Some("Enable/disable the nps shuttering banner and subpage")
+case object ShowNpsOutageBannerToggle extends FeatureFlagName {
```
Comment (pascal22p) [inline app/models/admin/FeatureFlags.scala:99]: rename ShowNpsOutageBannerToggle to ShowOutageBannerToggle

**Review by pascal22p — CHANGES_REQUESTED** [review]

File: app/models/admin/FeatureFlags.scala (hunk 4 above)
Comment (SandeepUmredkar) [reply app/models/admin/FeatureFlags.scala:99]: Done

File: app/models/admin/FeatureFlags.scala (hunk 3 above)
Comment (SandeepUmredkar) [reply app/models/admin/FeatureFlags.scala:165]: AddressJourneyEnforcerToggle is the replacement for NpsOutageToggle in the addressJourneyEnforcer. AddressChangeAllowedToggle is the toggle added for the change link to be shown or hidden

File: app/models/admin/FeatureFlags.scala (hunk 2 above)
Comment (SandeepUmredkar) [reply app/models/admin/FeatureFlags.scala:147]: This journey is to allow the user to edit the address or to show that address is changed.

File: app/models/admin/FeatureFlags.scala (hunk 3 above)
Comment (SandeepUmredkar) [reply app/models/admin/FeatureFlags.scala:165]: And When all is well this one is also set to true not false.

**Review by pascal22p — DISMISSED** [review]

**Review by pascal22p — DISMISSED** [review]

**Review by mathivanan-kailasam — APPROVED** [review]

**Pull request merged**